Example/Emission/AppDelegate.m

```diff
@@ -88,12 +90,20 @@ - (void)setupEmission;
 
   emission.switchBoardModule.presentNavigationViewController = ^(UIViewController * _Nonnull fromViewController,
                                                                  NSString * _Nonnull route) {
+    if (self.isShowingStorybook) {
+      NSLog(@"Route push - %@", route);
+      return;
+    }
     [fromViewController.navigationController pushViewController:[self viewControllerForRoute:route]
                                                        animated:YES];
   };
 
   emission.switchBoardModule.presentModalViewController = ^(UIViewController * _Nonnull fromViewController,
                                                             NSString * _Nonnull route) {
+    if (self.isShowingStorybook) {
+      NSLog(@"Route modal - %@", route);
+      return;
+    }
```

Comment:
I’m not sure I understand why this wouldn’t work?

```objc
if ([fromViewController isKindOf:ARStorybookComponentViewController.class]) {
  NSLog(@"Route modal - %@", route);
  return;
}
```

My main preference for this is that it serves the purpose of an ‘example’ app, i.e. shows/tests the API.